### PR TITLE
Docs - update README with a more correct installation message

### PR DESCRIPTION
**Pull request by stijnmoreels**
Update README with a more correct installation message with the available common packages.

Relates to https://github.com/arcus-azure/arcus/issues/92

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,22 @@ Azure Event Grid development in a breeze.
 
 # Installation
 
+Easy to install it via NuGet:
+
+- **EventGrid publishing**
```
Comment (tomkerkhove) [inline README.md:13]:
```suggestion
- **Event Grid publishing**
```

Comment (tomkerkhove) [reply README.md:13]: Or just publishing?

File: README.md [hunk 2]
```diff
@@ -8,10 +8,22 @@ Azure Event Grid development in a breeze.
...
+
 ```shell
-PM > Install-Package Arcus.EventGrid.All
+PM > Install-Package Arcus.EventGrid.Publishing
 ```
 
+- **EventGrid models**
```
Comment (tomkerkhove) [inline README.md:19]:
```suggestion
- **Event Grid models**
```

Or just models?

**Review by tomkerkhove — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (stijnmoreels) [reply README.md:13]: Ah, yes, best just publishing.

File: README.md (hunk 2 above)
Comment (stijnmoreels) [reply README.md]: Ah, yes, best just models.

**Pull request merged**